openml/datasets/functions.py

```diff
@@ -333,6 +333,27 @@ def _load_features_from_file(features_file: str) -> Dict:
         return xml_dict["oml:data_features"]
 
 
+def _expand_parameter(parameter: Union[str, list]):
```

Comment:
```suggestion
def _expand_parameter(parameter: Union[str, List]) ->List[str]:
```

- `List` should be used as a type hint instead of `list` (`list` is only supported for type hints in Py3.9+)
- type hints include a return value (`-> None` of no return value)